docs/src/03-metadata-analysis.jmd

```diff
@@ -86,6 +97,12 @@ sort!(mgxsamples, :studyID);
 first(mgxsamples, 5) |> randrows
 ```
 
+**Is it worth noting that after a certain batch (will have to check exact #) we
```

Comment:
Not sure exactly what you mean, but if you think it's worth clarifying go ahead and write it up here.